content/kubermatic/master/tutorials_howtos/operating_system_manager/_index.en.md

```diff
@@ -71,7 +70,32 @@ More work is being done to make it even easier to use OSM in air-gapped environm
 | Openstack | ✓ | ✓ | ✓ | x |
 | VSphere | ✓ | ✓ | ✓ | x |
 
+## Machine rotation
+
+When there is a change in the Machine Deployment template then the Machine set and in-turn the machines are rotated; nodes are re-created. OSM will take care of re-generating the OperatingSystemConfigs and provisioning secrets when there is a change in the Machine Deployment template. So the newly created machines will pick up the new configs.
+
+Although, if an `OperatingSystemProfile` is updated then it wouldn't result in an automatic rotation of the machines. This is an intentional design decision since an `OperatingSystemProfile` can be associated with multiple Machine Deployments and it is the user's responsibility to rotate the machines when the `OperatingSystemProfile` is updated.
```

Comment:
```suggestion
However, if an `OperatingSystemProfile` is updated, that wouldn't result in an automatic rotation of the machines. This is an intentional design decision since an `OperatingSystemProfile` can be associated with multiple Machine Deployments and it is the user's responsibility to rotate the machines when the `OperatingSystemProfile` is updated.
```